src/viam/rpc/server.py

```diff
@@ -114,3 +134,21 @@ async def create_and_serve(
         """
         server = cls(components)
         await server.serve(host, port, log_level, path=path)
+
+
+def wrapper(func: Callable):
```

Comment:
[nit]
```suggestion
def error_wrapper(func: Callable):
```
```suggestion
def error_wrapping_wrapper(func: Callable):
```
```suggestion
def grpc_error_wrapper(func: Callable):
```
```suggestion
def wrap_with_grpc_error(func: Callable):
```
...? Not sure if any of these are perfect but a bit more description could help make this more obvious

Reply:
I agree with this. I'll hold off to see if the others have a preferred name.

Reply:
I like `grpc_error_wrapper`.